librz/type/path.c

```diff
@@ -291,6 +386,11 @@ RZ_API ut64 rz_type_db_struct_member_packed_offset(RZ_NONNULL const RzTypeDB *ty
 	rz_return_val_if_fail(typedb && name && member, 0);
 	RzBaseType *btype = rz_type_db_get_base_type(typedb, name);
 	if (!btype || btype->kind != RZ_BASE_TYPE_KIND_STRUCT) {
+		if (!btype) {
+			eprintf("return offset 0: !btype\n");
+		} else {
+			eprintf("return offset 0: btype->kind != RZ_BASE_TYPE_KIND_STRUCT\n");
+		}
```

Comment:
if this should be surfaced to users, it's better to use `RZ_LOG_ERROR` or similar.

Reply:
Nope, it shouldn't :) 
Deleted.